src/Type/ApiTypes.ts

```diff
@@ -309,3 +321,21 @@ export interface WorkOrderResponse {
     WorkOrderPayload: WorkOrderPayload
   }
 }
+
+export interface ContentData {
+  name: string
+  id: number
+  label: undefined

```

Comment:
same here , why undefined only ?

Reply:
i don't know the type actually, getting null in response ,changing into string for now, will change later once i got value in response